workflow/util/util.go

```diff
@@ -444,7 +444,10 @@ func updateWorkflowNodeByKey(wfIf v1alpha1.WorkflowInterface, hydrator hydrator.
 	return err
 }
 
-const letters = "abcdefghijklmnopqrstuvwxyz0123456789"
+const (
+	letters                      = "abcdefghijklmnopqrstuvwxyz0123456789"
+	previousWorkflowNameLabelKey = "workflows.argoproj.io/previous-workflow-name"
```

Comment:
This should be added in https://github.com/argoproj/argo/blob/master/workflow/common/common.go

Also I would suggest this be called
```suggestion
	previousWorkflowNameLabelKey = "workflows.argoproj.io/resubmitted-from-workflow"
```